masterlist.yaml

```
@@ -16720,3 +16720,22 @@ plugins:
           - 'Whiterun Expansion SSE - NON Open Cities Version'
           - '[Bells of Skyrim Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/34472/)'
         condition: 'active("WhiterunAddition.esp") and not active("Bells of Skyrim-Whiterun Expansion Patchs.esp")'
+
+  ### Majestic Mountains - Pilgrims Delight Patch ###
+  - name: MM_PD_Patch.esp
```

File name names should be wrapped in single quotes.
```suggestion
  - name: 'MM_PD_Patch.esp'
```